Problem with latest cmake config

Hi! I have to build `ros2_control` from source for a debian 11 system. Since the changes of this repo from last week I get the following errors from colcon, despite colcon reporting a successful build of `backward_ros`:

```
colcon     build     --mixin release build-testing-off     --cmake-args --no-warn-unused-cli     --packages-up-to ros2_control
...
#25 9.790 Finished <<< backward_ros [3.16s]
#25 157.4 Starting >>> controller_manager
#25 126.0 --- stderr: controller_manager
#25 126.0 CMake Error at CMakeLists.txt:22 (find_package):
#25 126.0   By not providing "Findbackward_ros.cmake" in CMAKE_MODULE_PATH this project
#25 126.0   has asked CMake to find a package configuration file provided by
#25 126.0   "backward_ros", but CMake did not find one.
#25 126.0 
#25 126.0   Could not find a package configuration file provided by "backward_ros" with
#25 126.0   any of the following names:
#25 126.0 
#25 126.0     backward_rosConfig.cmake
#25 126.0     backward_ros-config.cmake
#25 126.0 
#25 126.0   Add the installation prefix of "backward_ros" to CMAKE_PREFIX_PATH or set
#25 126.0   "backward_ros_DIR" to a directory containing one of the above files.  If
#25 126.0   "backward_ros" provides a separate development package or SDK, be sure it
#25 126.0   has been installed.
#25 126.0 
#25 126.0 
#25 126.0 ---
#25 158.7 Failed   <<< controller_manager [1.29s, exited with code 1]
```

Is this a problem with the new cmake config here, or am I doing something wrong in my build?
With the version 1.0.2 I don't have any problems with building from source or released packages on Ubuntu.

I think with ros-controls we don't build backward_ros from source, this is why we haven't seen the error there in the CI yet.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,6 +129,7 @@ configure_package_config_file(
 
 configure_file(cmake/BackwardConfig.cmake ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake @ONLY)
 
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake DESTINATION share/${PROJECT_NAME}/cmake)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake DESTINATION share/${PROJECT_NAME}/cmake)
 
 include(CTest)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,7 +129,7 @@ configure_package_config_file(
 
 configure_file(cmake/BackwardConfig.cmake ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake DESTINATION share/${PROJECT_NAME}/cmake)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake DESTINATION share/${PROJECT_NAME}/cmake)
 
 include(CTest)
 if(BUILD_TESTING)

